lms/templates/problem.html

```diff
@@ -29,7 +29,7 @@ <h2 class="problem-header">
     <button class="save" data-value="${_('Save')}">${_('Save')}<span class="sr"> ${_("your answer")}</span></button>
     % endif
     % if answer_available:
-    <button class="show"><span class="show-label">${_('Show Answer')}</span> </button>
+    <button class="show"><span class='sr'>${_('Toggle Answer Visibility')}</span><span class="show-label">${_('Show Answer')}</span></button>
```

Comment:
@clytwynec I like the additional context here.  :+1: It's the simplest solution ATM